clients/drcachesim/tracer/raw2trace.h

```diff
@@ -1424,11 +1434,10 @@ template <typename T> class trace_converter_t {
                     // not-inside-a-block main loop.
                 }
             } else {
-                // Other than kernel event markers checked above, markers should be
-                // only at block boundaries, as we cannot figure out where they should go
-                // (and could easily insert them in the middle of this block instead
-                // of between this and the next block, with implicit instructions added).
-                // Thus, we do not append any other markers.
+                // All markers should be only at block boundaries, as we cannot
+                // figure out where they should go (and could easily insert them in the
+                // middle of this block instead of between this and the next block, with
+                // implicit instructions added). Thus, we do not append any other markers.
```

Comment:
I think my prior comment was asking to explain here and in the top-level comment for this function that *all* intra-block markers are inserted by this function (i.e., the higher level process_offline_entry() will never insert a marker intra-block) and all inter-block markers are handled at a higher level (process_offline_entry()) and are never inserted here.

Reply:
Added this at the top-level comment for the function.